charts/images.yaml

```diff
@@ -41,8 +41,8 @@ images:
 - name: csi-provisioner
   sourceRepository: github.com/kubernetes-csi/external-provisioner
   repository: registry.k8s.io/sig-storage/csi-provisioner
-  tag: v3.5.0
-  targetVersion: ">= 1.24"
+  tag: v3.6.0
+  targetVersion: ">= 1.22"
```

Comment:
Why was this changed back, With https://github.com/gardener/gardener-extension-provider-vsphere/pull/472, we don't support Shoots < 1.24.